core/Pos/Core/Genesis.hs

```diff
@@ -93,7 +93,6 @@ genesisStakeDistribution
     | isDevelopment = def
     | otherwise     = gcdDistribution compileGenCoreData
 
-genesisBalances :: HashMap StakeholderId Coin
-genesisBalances
-    | isDevelopment = mempty
-    | otherwise     = gcdBootstrapBalances compileGenCoreData
+-- | Genesis stakes which should be used only in production mode.
+genesisStakesProdMod :: StakesMap
```

Comment:
1. Previously one could accidentally pass `genesisBalances` somewhere in development mode, but empty genesis stakes don't make sense at all (they make system broken). Now name makes it clear that they shouldn't be used in development mode.
2. Name `Balances` is wrong.
3. Using type aliases is a good practice.
4. I want to get rid of `isDevelopment` in as many places as possible. It's not my current goal, but when I see such opportunity, I do it.